dfastmi/io/CelerObject.py

```diff
@@ -37,36 +37,52 @@
     CelerProperties
 
 """
-from abc import ABC, abstractmethod
-from typing import List
+from abc import ABC
+from typing import List, Tuple
+from pydantic import BaseModel
+from dfastmi.io.AReach import AReach
 
-
-class ICelerObject(ABC):
+class ICelerObject(ABC, BaseModel):
```

Comment:
Interfaces in Python aren't really a thing. Python allows for multi inheritance, and you should use abstract or base classes for this.
It might sound like potato potato, but I think it is important to adhere to a certain standard.